.github/workflows/build-test-windows.yml

```diff
@@ -0,0 +1,33 @@
+
+name: build-and-test-windows
+on:
+  push:
+    branches: [ main ]
```

Comment:
The main branch for this repo is still `master`. Smells like a copy pasta mistake

Reply:
I have no idea how this is even working!  wow...

Reply:
I think it's because the `pull_request` caught this PR. The push would be after this gets merged

Reply:
I'll change it, but also longer term need to move to main branch as default.